koku/masu/util/gcp/common.py

```diff
@@ -191,9 +200,15 @@ def match_openshift_resources_and_labels(data_frame, cluster_topology, matched_t
 
     cluster_id = cluster_topology.get("cluster_id")
     cluster_alias = cluster_topology.get("cluster_alias")
+    resource_ids = cluster_topology.get("resource_ids", [])
```

Comment:
Here we actually want nodes. 
```suggestion
    resource_ids = cluster_topology.get("nodes", [])
```

Reply:
And although we haven't quite confirmed that volumes match, it would not hurt to match what we're doing for Azure here to also check for matching volumes